packages/@lwc/engine/src/framework/services.ts

```diff
@@ -62,8 +62,8 @@ export function invokeServiceHook(vm: VM, cbs: ServiceCallback[]) {
             `Optimize invokeServiceHook() to be invoked only when needed`
         );
     }
-    const { component, data, def, context } = vm;
```

Comment:
I'm good with this as it is... but eventually we should just remove all services now that wire reform is in. The services are NOT used by anyone else ATM AFAIK cc @jodarove